src/main/java/domain/GameSystem.java

```diff
@@ -0,0 +1,53 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import utils.RandomUtils;
+import view.InputView;
+import view.OutputView;
+
+public class GameSystem {
+
+    final static private int START_RANDOM_NUM = 0;
+    final static private int END_RANDOM_NUM = 10;
+
+    public static List<Car> makeCarList(String[] splittedCarString) {
+        List<Car> carList = new ArrayList<Car>();
```

Comment:
```suggestion
        List<Car> carList = new ArrayList<>();
```
앞쪽에서 타입을 명시해줬다면 생성하는 쪽에서는 써주지 않아도 됩니다.
(JAVA 8 타입추론)

Reply:
처음에는 타입을 안적어줬었습니다. 그런데 인텔리제이에서 실행은 되는데 java uses unchecked or unsafe operations 라는 경고를 터미널에서 찍어내길래 https://www.oops4u.com/2240 이 블로그에서 타입을 적어줘야 한다고 한다길래 적고 해결이 되었습니다.

Reply:
자바 8 버전보다 낮은 버전을 사용했으면 그럴 수 있습니다
그렇지만 자동으로 설정되었다면 8 이상일테니 그럴 가능성은 적고
아마 List carList = new ArrayList<>(); 이런식으로 사용하지 않았을까? 생각해봐요
또는 List carList = new ArrayList();

Reply:
아 맞습니다!  List carList = new ArrayList(); 이렇게 썼던거 같아요!